### DISCUSSION TITLE
Don't power down or sleep between timelapse intervals in order to transfer files to Laptop.

### DISCUSSION THREAD

**Discussion by VreekFonq** [gopro/labs#1136] [Q&A]
Hi,
We want to setup 3 Go Pro angles with the Hero 12 for a year run of a time-lapse (individual images every 5 minutes) on a construction site. The setup will be powered via mains supply and USB adapter with PD. Battery removed from Go Pro 12.

We tested with hookup to USB PD hub and connected to a laptop for image download (no WiFi or Cloud upload possible). Problem is the shutdown between images. Laptop can only transfer media files when turned on. But we want to transfer the images any time of the day when possible, while the time-lapse is running.

Any ideas?
Thanks in advance,
Regards, Ramon

**Comment by dnewman-gpsw** [comment]
It is not possible to do that currently, it is the shutdowns that are providing all the power benefits. There is no variable state Labs can see between USB power connected, and USB data connected. You will have to stop or pause the script. 

P.S. you should keep the battery in for the best reliability. I've done a 6 month long H12 timelapse with the battery in, I haven't tested that length with the battery out.  You don't need to worry about heat, as the camera is mostly off.
